R/verb-select.R

```diff
@@ -101,6 +101,10 @@ simulate_vars <- function(x) {
 # op_select ---------------------------------------------------------------
 
 op_select <- function(x, vars) {
+  if (can_select_return_early(x, vars)) {
```

Comment:
Again, I think I'd rather give this a more descriptive name, like `selects_same_variables()` maybe?